analysis/include/RawSvtHitHistos2.h

```diff
@@ -0,0 +1,48 @@
+#ifndef RAWSVTHITHISTOS_H
+#define RAWSVTHITHISTOS_H
+
+#include "TFile.h"
+#include "HistoManager.h"
+#include "TGraphErrors.h"
+#include "TKey.h"
+#include "TList.h"
+#include "TH1.h"
+#include "RawSvtHit.h"
+//include "BaseSelectorSVT.h"
+//#include "AnaHelpers.h"
+
+//#include "BaseSelector.h"
+//#include "AnaHelpers.h"
+
+#include "ModuleMapper.h"
+
+#include <string>
+
+
+class RawSvtHitHistos2 : public HistoManager{
```

Comment:
Why are there two RawSvtHitHistos classes?

Reply:
Was a very early thing that I was doing to get RawSvtHitAnaProcessor working. Never needed it. Has been removed.

Reply:
I believe that that is the only such instance but will read it over carefully.